drivers/serial/Kconfig.xen

```diff
@@ -0,0 +1,30 @@
+# Xen hypervisor console via UART setup
+#
+# Copyright (c) 2021 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config UART_XEN_HVC
+	bool "Xen hypervisor console UART Driver"
+	select SERIAL_HAS_DRIVER
+	depends on BOARD_XENVM
+	default y
+	help
+	  Enable Xen hypervisor console driver.
+
+config XEN_HVC_INIT_PRIORITY
+	int "Xen hypervisor console init priority"
+	depends on UART_XEN_HVC
+	default 55
+	help
+		Set init priority for Xen HVC, should be inited before UART
+		console driver (HVC get inited on PRE_KERNEL_1 stage).
```

Comment:
indentation. Same for the next symbol help. Also s/get/gets/.

Reply:
Fixed.